src/assets/properties-panel.css

```diff
@@ -1097,10 +1097,6 @@ textarea.bio-properties-panel-input {
   padding: 2px 6px;
 }
 
-.bio-properties-panel-feel-editor-container .cm-scroller {
-  overflow: hidden !important;
```

Comment:
@marstamm I drilled as far as I could in the history and from what I've seen you added these styles. I couldn't notice anything this was fixing, but maybe I missed something and you remember why this was necessary

Reply:
Happy to merge if it also works in feel editors embedded inside the properties panel. There we want the editor to always have full height.

Reply:
@nikku is this style affecting the native property panel fields? Then we should probably find a dedicated fix in the popup editor

Reply:
> @nikku is this style affecting the native property panel fields? 

I don't know, but I gave this pointer to @vsgoulart to be able to assess the end-to-end impact of removing a prior fix (again).